src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1551,7 +1551,14 @@ static Library()
             },
             {
                 BuiltinFunctionsCore.CharT,
-                StandardErrorHandlingTabularOverload<NumberValue>(BuiltinFunctionsCore.CharT.Name, SimpleFunctionImplementations[BuiltinFunctionsCore.Char])
+                StandardErrorHandlingTabularOverload<NumberValue>(BuiltinFunctionsCore.CharT.Name, StandardErrorHandling<NumberValue>(
+                    BuiltinFunctionsCore.Char.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: ReplaceBlankWithZero,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: Char))
```

Comment:
NOTE: Since now we are removing Blank Handling from scaler overload, Tabular overload need to specify this explicitly.